viewer/jest.config.js

```diff
@@ -38,6 +38,7 @@ module.exports = () => {
     automock: false,
     setupFiles: [path.resolve(__dirname, './test-utilities/src/setupJest.ts'), 'jest-canvas-mock'],
     setupFilesAfterEnv: ['jest-extended/all'],
-    testEnvironment: 'jsdom'
+    testEnvironment: 'jsdom',
+    workerIdleMemoryLimit: '512MB'
```

Comment:
Will it affect our CI time or something related?

Reply:
It actually didn't, surprisingly. The last check I did ran about 1 minute (8 vs 9 minutes) faster than the other PR I checked. So not worried about that